movement/io/load_poses.py

```diff
@@ -678,3 +679,102 @@ def _ds_from_valid_data(data: ValidPosesDataset) -> xr.Dataset:
             "source_file": None,
         },
     )
+
+
+def from_mmp_file(
+    file_path: Union[Path, str], fps: Optional[float] = None
+) -> xr.Dataset:
+    """Load pose tracking data from a JSON file with multiple individuals.
+
+    This function expects JSON data in the following format:
```

Comment:
```suggestion
        Path to the .json file containing the pose tracking data.
```
For consistency, because we use .lowercase for file extensions throughout this module (.csv, .h5 etc.)